### PR TITLE
Update dapp tests to be compatible with test chains

### PR DESCRIPTION
**Pull request by mj850**
## Describe your changes and provide context
We want to occasionally be able to run our suite of tests on devnet and testnet.

This changes makes the tests in integration_test/dapp_tests runnable on devnet and testnet and adds a github actions pipeline to allow devs to run those tests on testnet/devnet on demand.

The tests rely on the test address 0xC56B90739aFF8e718516b38a59D9DD236cCAFd3b being funded with sufficient Sei.
## Testing performed to validate your change
Testing on new test pipeline
Most recent run:

https://github.com/sei-protocol/sei-chain/actions/runs/10411442874

### CODE REVIEW

File: integration_test/dapp_tests/steak/SteakTests.js [hunk 1]
```diff
@@ -14,38 +16,45 @@ const {
...
       await associateKey(account.address);
     }
+
     return account;
   }
 
   async function deployContracts(ownerAddress) {
     // Store CW20 token wasm
-    const tokenCodeId = await storeWasm(STEAK_TOKEN_WASM);
+    const STEAK_TOKEN_WASM = (await isDocker()) ? '../integration_test/dapp_tests/steak/contracts/steak_token.wasm' : path.resolve(__dirname, '../steak/contracts/steak_token.wasm')
```
Comment (besated) [inline integration_test/dapp_tests/steak/SteakTests.js:52]: nice thanks for updating this!

File: loadtest/loadtest_client.go [hunk 2]
```diff
@@ -265,7 +265,7 @@ func (c *LoadTestClient) SendTxs(
 				fmt.Printf("Failed to acquire semaphore: %v", err)
 				break
 			}
-			if tx.TxBytes != nil && len(tx.TxBytes) > 0 {
+			if len(tx.TxBytes) > 0 {
```
Comment (besated) [inline loadtest/loadtest_client.go:268]: is this intentional? seems unrelated

Comment (mj850) [reply loadtest/loadtest_client.go:268]: Ah yup this was causing a lint issue that was blocking the PR. Tony makes the same change in his PR here: https://github.com/sei-protocol/sei-chain/pull/1818/files

**Review by besated — APPROVED** [review]
